docs/csharp/how-to/modify-string-contents.md

```diff
@@ -0,0 +1,76 @@
+---
+title: "How to: Modify String Contents (C# Guide)"
+ms.date: 02/26/2018
+ms.prod: .net
+ms.technology: 
+  - "devlang-csharp"
+ms.topic: "article"
+helpviewer_keywords: 
+  - "strings [C#], modifying"
+author: "BillWagner"
+ms.author: "wiwagn"
+---
+
+# How to: Modify String Contents (C# Programming Guide)
+
+This article demonstrates several techniques to produce a `string` by modifying an existing `string`. All the techniques demonstrated return the result of the modifications as a `string` object. To clearly demonstrate this, the examples all store the result in a new variable. You can then examine both the original `string` and the `string` resulting from the modification when you run each example. 
+
+There are several techniques demonstrated in this article. You can replace existing text. Other methods search for patterns and replace matching text with other text. You can also treat a string as a sequence of characters. You can also use convenience methods that remove whitespace. YOu should choose the techniques that most closely matches your scenario.
```

Comment:
Rather than "Other methods", "You" to maintain parallelism
No "also" in "You can also treat"
whitespace --> white space (2 words)
YOu has a typo --> You
matches ---> match